tests/unit/data/_async/test_client.py

```diff
@@ -36,11 +36,15 @@
 
 from google.cloud.bigtable.data._cross_sync import CrossSync
 from tests.unit.data.execute_query.sql_helpers import (
+    chunked_responses,
```

Comment:
are there any system tests that can be added?

Reply:
I have a follow on with a couple of new integration tests. W.r.t the chunked responses there isn't a great way to verify if the backend is sending multiple chunks or a single chunk.

I do have a significant number of conformance tests ready to go for this logic though, and im planning on adding the testproxy support for python as a fast follow